src/pages/People.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable max-len */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import PeopleTable from '../components/PeopleTable/PeopleTabe';
+import { Person } from '../types';
+
+function PeoplePage() {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    getPeople()
+      .then((data) => {
+        setPeople(data);
+        setLoading(false);
+      })
+      .catch((err) => {
+        setError(err.message);
+        setLoading(false);
+      });
```

Comment:
I think `setLoading` should be set in `.finally`

Reply:
Of course it should. Thanks!